examples/Forms/TextInput.md

```diff
@@ -0,0 +1,128 @@
+The following is an example of the TextInput component that includes the optional `errorMessage` property. 
+```jsx
+  import { useState } from 'react';
+  const TextInputExample = () => {
+    const [value, setValue] = useState('Invalid value');
+    return (<TextInput
+      value={value}
+      name="example"
+      id="example"
```

Comment:
All our examples should be valid code, so we need a `label` value here

Reply:
Fixed Text Input example through fc997fe